modules/contentbox/modules/contentbox-admin/modules/contentbox-filebrowser/handlers/Home.cfc

```diff
@@ -456,7 +456,7 @@ component{
 		}
 		// Manual uploader?
 		if( rc.manual AND !data.errors) {
-			event.renderData( data="<textarea id='data_result'='upload'>#serializeJSON( data )#</textarea>", type="text" );
+			event.renderData( data=serializeJSON( data ), type="text" );
```

Comment:
Come to think of it, this can be just removed and just remove the if statement and leave:


```coldfusion
event.renderData( data=data, type="json" );
```

Reply:
@lmajano I did that but then the I received a console warning because it received the document as a JSON format instead of a document format.

Reply:
hmm, which part? the manual part?

Reply:
Ohh, maybe the iframe part right?

Reply:
@lmajano Yes.